drivers/Aqara/aqara-lock/src/init.lua

```diff
@@ -67,41 +69,60 @@ local function toHex(value, length)
 end
 
 local function event_lock_handler(driver, device, evt_name, evt_value)
-  if evt_value == 1 then
+  if evt_value == 0x1 then
     device:emit_event(Lock.lock(evt_name))
-    device:emit_event(TamperAlert.tamper.clear())
+    device:emit_event(LockAlarm.alarm.clear({ visibility = { displayed = false }}))
     remoteControlShow(device)
   end
 end
 
 local function event_unlock_handler(driver, device, evt_name, evt_value)
   local id, label
+  if evt_value == 0x80020000 then -- one-time password
+    id = "OTP_STANDALONE"
+    label = nil
+  else
   id, label = credential_utils.find_userLabel(driver, device, evt_value)
+  end
   device:emit_event(Lock.lock.unlocked({ data = { method = evt_name, codeId = id, codeName = label } }))
   device:emit_event(remoteControlStatus.remoteControlEnabled('false', { visibility = { displayed = false } }))
-  device:emit_event(TamperAlert.tamper.clear())
+  device:emit_event(LockAlarm.alarm.clear({ visibility = { displayed = false }}))
```

Comment:
same here

Reply:
@seojune79 Please review when your schedule allows.

Reply:
@greens @lelandblue , please visit `STSE-2969` for details.